### PR TITLE
node v14 compatibility pass

### PR DESCRIPTION
**Pull request by derek-oneill**

### CODE REVIEW

**Comment by derek-oneill** [comment]
associated PR https://github.com/entrinsik-org/i5/pull/4636

File: lib/resource.js [hunk 1]
```diff
@@ -4,6 +4,7 @@ var fs = require('fs');
 var vfs = require('vinyl-fs');
 var path = require('path');
 var su = require('./stream-utils');
+const { paths } = require('./stream-utils');
```
Comment (gmsewell6) [inline lib/resource.js:7]: unnecessary. already used above

File: lib/resource.js [hunk 2]
```diff
@@ -57,13 +58,18 @@ Resource.prototype.minify = function (opts, callback) {
...
         .pipe(su.paths('/'+ base, opts.checksums, function (err, paths) {
             self.paths = paths;
         }))
         .on('end', function() {
             callback(null, self);
         });
+    }
+    else {
+        callback(null, this);
+    }
```
Comment (gmsewell6) [inline lib/resource.js:72]: how is this necessary for node-14 compatibility

**Review by gmsewell6 — CHANGES_REQUESTED** [review]
see comments

Also simply running npm install rearranges package.json properly, which currently results in an always-modified version because this PRs is out of order. actually run npm install and let it fix the package.json contents organization before committing

File: lib/resource.js (hunk 2 above)
Comment (derek-oneill) [reply lib/resource.js:72]: If paths is empty you get vfs errors on startup.

**Pull request merged**